src/module-init/module-init.c

```diff
@@ -116,6 +101,56 @@ static int initAsLibrary(RedisModuleCtx *ctx) {
 int RS_Initialized = 0;
 RedisModuleCtx *RSDummyContext = NULL;
 
+static int HashNotificationCallback(RedisModuleCtx *ctx, int type, const char *event, RedisModuleString *key) {
```

Comment:
I do not think this should be here, where did we decided to put it?

Reply:
@MeirShpilraien  I'll create a new file `src/notifications.c`. ok?